TrkFilters/src/KalSeedFilter_module.cc

```diff
@@ -168,14 +171,17 @@ namespace mu2e
             if(!_noInfo)triginfo->_tracks.push_back(art::Ptr<KalSeed>(ksH,index));
 
             if(_debug > 1){
-              std::cout << moduleDescription().moduleLabel() << " passed event " << evt.id() << std::endl;
+              std::cout << moduleDescription().moduleLabel() << " --> accepted a track\n";
```

Comment:
Is there a reason you switched to \n instead of endl?

Reply:
I typically use `\n` if I'm using text at the end of the line, and use `<< std::endl` otherwise (when they're treated similarly, at least). I can switch it back here though as it wasn't a motivated choice, I was just clarifying a track was accepted vs. the event and printing the event ID once.